docs/UserGuide.md

```diff
@@ -31,104 +31,238 @@
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
+#### Deleting a task: `delete`
+Deletes the selected task from Lifekeeper. Irreversible.<br>
 Format: `delete INDEX`
 
-> Deletes the person at the specified `INDEX`. 
+> Deletes the task with `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+  
 
 Examples: 
 * `list`<br>
+  `delete 1,3`<br>
+  Deletes the 1st and 3rd task in the Lifekeeper task list.
+* `find Dinner`<br>
   `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  Selects the 2nd task in the results of the `find` command and then deletes it.
+
+#### Select: `select`
+Category
+Selects the category identified by the index number used in the last task listing.<br>
+Format: `select INDEX`
+
+> Selects the category at the specified `INDEX`. 
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
+
+Examples: 
+* `list`<br>
+  `select 2`<br>
+  Selects the 2nd category in category list.
+
+Task
+Selects the task identified by the index number used in the last task listing.<br>
 Format: `select INDEX`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
+> Selects the task at the specified `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
+
 Examples: 
 * `list`<br>
   `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
+  Selects the 2nd task in Lifekeeper.
+* `find Dinner` <br> 
   `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+  Selects the 1st task in the results of the `find` command.
+* `list category` <br> 
+  `select 3`<br>
+  `select 1`<br>
+  Selects the 3rd category in the list and then select the 1st task shown in the category..
+
+#### Marking a task as done: `done`
+Marks the task as completed.<br>
+Format: `done INDEX`
+
+> Marks the task with `INDEX` as completed.
+  The index refers to the index number shown in the most recent listing.<br>
+  
+
+Examples: 
+* `list`<br>
+  `done 1`<br>
+  Marks the 1st task in the Lifekeeper task list as completed.
+* `find Admin`<br>
+  `done 2`<br>
+  Selects the 2nd task in the results of the `find` command and then marks it as completed.
+
+#### Editing a task: `edit`
+Edits the selected task from Lifekeeper. Irreversible.<br>
+Format: `edit [TASK_NAME] [c/CATEGORY] [d/DEADLINE] p/PRIORITY_LEVEL r/REMINDER [t/TAG]...`
+
+> Edits the task that was previously selected with `INDEX`. 
```

Comment:
generally, we encourage you to have "one shot" commands. to edit you need to have selected the task which is unnecessary.
try using the index to perform the edit in one shot.